### DISCUSSION TITLE
Programming a M27C160

### DISCUSSION THREAD

**Discussion by TheRealNextria** [sanni/cartreader#230] [Need help]
Hello Sanni,

Is it possible to also program a M27C160 eeprom?
I see the M27C322 is supported.

would be great to program Sega saturn bios chips.

**Comment by sanni** [comment]
I did not test it but it could work. 
Obviously you would need to set the two switches on the Eprom adapter to match the 27C160 pinout and change the flashSize variable in the code to whatever the size of the 27C160 is.
